samples-final/8.AspNetCore-LUIS-Bot/MyLuisBot.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Ai.Luis;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Schema;
+
+namespace AspNetCore_LUIS_Bot
+{
+    public class MyLuisBot : IBot
+    {
+        private const double LUIS_INTENT_THRESHOLD = 0.2d;
+
+        private readonly MyBotAccessors _stateAccessors;
+        private readonly LuisRecognizer _luisRecognizer;        
+        private readonly DialogSet _dialogSet;
+
+        public MyLuisBot(MyBotAccessors accessors, LuisRecognizer recognizer)
+        {
+            _stateAccessors = accessors ?? throw new ArgumentNullException(nameof(accessors));
+            _luisRecognizer = recognizer ?? throw new ArgumentNullException(nameof(recognizer));
+
+            _dialogSet = new DialogSet();
+
+            var reminderDialog = new ReminderDialog(_stateAccessors);            
+            _dialogSet.Add("AlarmDialog", reminderDialog); 
```

Comment:
We should have a consistent name here. Typo - alarmDialog and reminderDialog

Reply:
Fixed